### ISSUE TITLE
remix hangs : 0.4.13

### ISSUE THREAD

**Issue by GoldenDave** [ethereum/browser-solidity#628]
```
pragma solidity ^0.4.13;

contract test{
    
    uint256 rateN = 9999452054794520548;
    uint256 maxRate = 9999452054794520548;
    
    function testIt() returns (uint256 r) {
        uint256 pow = 2;
        uint256 newN = rateN ** pow; // check for overflow
        uint256 checkVal = newN / maxRate;
        r = checkVal;
    }
}
```

call testIt(), it hangs on `uint256 newN = rateN ** pow;`

**Comment by GoldenDave** [comment]
This Works
```
pragma solidity ^0.4.13;

contract test{
    uint256 rateMM = 9999452054794520548;
    uint256 rateN = 999945205479452;
    uint256 maxRate = 999945205479452;
    
    function testIt() returns (uint256 r) {
        uint256 pow = 2;
        uint256 newN = rateN ** pow; // check for overflow
        uint256 checkVal = newN / maxRate;
        r = checkVal;
    }
}
```

**Comment by axic** [comment]
I think this might be actually a bug in Remix. The `ISZERO` instruction according to the debugger is not just removing the top stack item.

@yann300 any ideas?

**Comment by yann300** [comment]
yeah I saw that ... I was about to take a look at the ISZERO implementation of ethereumjs-vm .. but no time to do that yet... could you take a short look ? you might know js-vm better than me.

**Comment by axic** [comment]
Reason is #634.

**Comment by axic** [comment]
Fixed by #635.
